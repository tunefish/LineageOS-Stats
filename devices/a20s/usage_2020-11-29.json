{
  "17.1-20200715-unofficial-tissot": 1268,
  "17.1-20200821-unofficial-z3": 919
}