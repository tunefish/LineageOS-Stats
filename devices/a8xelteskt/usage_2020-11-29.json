{
  "17.1-20200715-unofficial-tissot": 1257,
  "17.1-20200716-unofficial-herolte": 1,
  "17.1-20200821-unofficial-z3": 923
}