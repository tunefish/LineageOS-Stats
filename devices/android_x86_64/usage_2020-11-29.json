{
  "14.1-20170228-unofficial-android_x86_64": 4,
  "14.1-20180821-unofficial-android_x86_64": 1,
  "14.1-20200916-unofficial-android_x86_64": 4,
  "14.1-20200921-zts-android_x86_64": 1,
  "14.1-20200922-zts-android_x86_64": 1,
  "14.1-20200923-aos-android_x86_64": 1,
  "14.1-20200923-zts-android_x86_64": 12,
  "14.1-20200927-aos-android_x86_64": 1,
  "14.1-20200928-aos-android_x86_64": 1,
  "14.1-20201009-unofficial-android_x86_64": 3,
  "14.1-20201009-zts-android_x86_64": 3,
  "14.1-20201010-aos-android_x86_64": 1,
  "14.1-20201010-unofficial-android_x86_64": 1,
  "14.1-20201012-unofficial-android_x86_64": 3,
  "14.1-20201014-unofficial-android_x86_64": 4,
  "14.1-20201015-unofficial-android_x86_64": 4,
  "14.1-20201016-unofficial-android_x86_64": 1,
  "14.1-20201017-unofficial-android_x86_64": 1,
  "14.1-20201021-unofficial-android_x86_64": 4,
  "14.1-20201021-zts-android_x86_64": 1,
  "14.1-20201022-unofficial-android_x86_64": 6,
  "14.1-20201111-aos-android_x86_64": 2,
  "14.1.0-android_x86_64": 19979,
  "android_x86_64-userdebug 10 qq3a.200805.001 eng.electr.20200828.100138 test-keys": 40,
  "swat2015-lineaeos14.1": 1
}