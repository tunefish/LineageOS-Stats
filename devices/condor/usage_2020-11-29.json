{
  "13.0-20170212-stable-condor": 7,
  "13.0-20170621-stable-condor": 24,
  "13.0-20171021-unofficial-condor": 11,
  "14.1-20170117-unofficial-condor": 2,
  "14.1-20170119-unofficial-condor": 38,
  "14.1-20170123-nightly-condor": 7,
  "14.1-20170130-nightly-condor": 7,
  "14.1-20170206-nightly-condor": 5,
  "14.1-20170213-nightly-condor": 14,
  "14.1-20170227-nightly-condor": 4,
  "14.1-20170306-nightly-condor": 3,
  "14.1-20170313-nightly-condor": 8,
  "14.1-20170320-nightly-condor": 10,
  "14.1-20170327-nightly-condor": 2,
  "14.1-20170403-nightly-condor": 6,
  "14.1-20170410-nightly-condor": 14,
  "14.1-20170424-nightly-condor": 4,
  "14.1-20170501-nightly-condor": 7,
  "14.1-20170508-nightly-condor": 9,
  "14.1-20170515-nightly-condor": 4,
  "14.1-20170522-nightly-condor": 6,
  "14.1-20170529-nightly-condor": 5,
  "14.1-20170605-nightly-condor": 12,
  "14.1-20170612-nightly-condor": 7,
  "14.1-20170620-nightly-condor": 7,
  "14.1-20170626-nightly-condor": 14,
  "14.1-20170703-nightly-condor": 13,
  "14.1-20170710-nightly-condor": 3,
  "14.1-20170717-nightly-condor": 13,
  "14.1-20170724-nightly-condor": 5,
  "14.1-20170731-nightly-condor": 5,
  "14.1-20170807-nightly-condor": 11,
  "14.1-20170814-nightly-condor": 8,
  "14.1-20170821-nightly-condor": 9,
  "14.1-20170828-nightly-condor": 6,
  "14.1-20170904-nightly-condor": 6,
  "14.1-20170911-nightly-condor": 7,
  "14.1-20170918-nightly-condor": 4,
  "14.1-20170925-nightly-condor": 5,
  "14.1-20171002-nightly-condor": 6,
  "14.1-20171009-nightly-condor": 6,
  "14.1-20171016-nightly-condor": 17,
  "14.1-20171023-nightly-condor": 14,
  "14.1-20171030-nightly-condor": 6,
  "14.1-20171106-nightly-condor": 13,
  "14.1-20171113-nightly-condor": 5,
  "14.1-20171120-nightly-condor": 5,
  "14.1-20171127-nightly-condor": 16,
  "14.1-20171204-nightly-condor": 3,
  "14.1-20171211-nightly-condor": 7,
  "14.1-20171218-nightly-condor": 7,
  "14.1-20171225-nightly-condor": 7,
  "14.1-20180101-nightly-condor": 10,
  "14.1-20180108-nightly-condor": 16,
  "14.1-20180122-nightly-condor": 4,
  "14.1-20180129-nightly-condor": 9,
  "14.1-20180205-nightly-condor": 10,
  "14.1-20180212-nightly-condor": 21,
  "14.1-20180220-nightly-condor": 14,
  "14.1-20180228-nightly-condor": 10,
  "14.1-20180307-nightly-condor": 8,
  "14.1-20180314-nightly-condor": 9,
  "14.1-20180321-nightly-condor": 17,
  "14.1-20180328-nightly-condor": 9,
  "14.1-20180404-nightly-condor": 3,
  "14.1-20180410-nightly-condor": 6,
  "14.1-20180418-nightly-condor": 11,
  "14.1-20180425-nightly-condor": 5,
  "14.1-20180503-nightly-condor": 9,
  "14.1-20180509-nightly-condor": 4,
  "14.1-20180516-nightly-condor": 9,
  "14.1-20180523-nightly-condor": 8,
  "14.1-20180530-nightly-condor": 9,
  "14.1-20180606-nightly-condor": 6,
  "14.1-20180613-nightly-condor": 14,
  "14.1-20180620-nightly-condor": 8,
  "14.1-20180627-nightly-condor": 23,
  "14.1-20180711-nightly-condor": 12,
  "14.1-20180718-nightly-condor": 6,
  "14.1-20180725-nightly-condor": 7,
  "14.1-20180801-nightly-condor": 7,
  "14.1-20180808-nightly-condor": 12,
  "14.1-20180815-nightly-condor": 2,
  "14.1-20180822-nightly-condor": 9,
  "14.1-20180829-nightly-condor": 4,
  "14.1-20180905-nightly-condor": 11,
  "14.1-20180912-microg-condor": 1,
  "14.1-20180912-nightly-condor": 9,
  "14.1-20180919-nightly-condor": 14,
  "14.1-20180926-nightly-condor": 5,
  "14.1-20181003-nightly-condor": 11,
  "14.1-20181010-nightly-condor": 11,
  "14.1-20181017-nightly-condor": 15,
  "14.1-20181024-nightly-condor": 14,
  "14.1-20181031-nightly-condor": 15,
  "14.1-20181107-nightly-condor": 28,
  "14.1-20181120-nightly-condor": 2,
  "14.1-20181121-nightly-condor": 53,
  "14.1-20181219-microg-condor": 1,
  "14.1-20181220-nightly-condor": 260,
  "14.1-20190102-microg-condor": 10,
  "14.1-20190409-unofficial-condor": 1,
  "14.1-20191113-unofficial-condor": 3,
  "14.1-20200308-unofficial-condor": 1,
  "14.1-20200408-unofficial-condor": 2,
  "14.1-20200419-unofficial-condor": 2,
  "14.1-20200611-unofficial-condor": 5,
  "14.1-20200724-unofficial-condor": 5,
  "14.1-20200825-unofficial-condor": 16,
  "14.1-20200925-ridebmx-condor": 2,
  "14.1-20201006-raoul-condor": 1,
  "14.1-20201023-ridebmx-condor": 1,
  "14.1-20201023-unofficial-condor": 34,
  "14.1-20201103-unofficial-condor": 1,
  "14.1-20201115-unofficial-condor": 1,
  "15.1-20180226-unofficial-condor": 4,
  "15.1-20180304-unofficial-condor": 325,
  "15.1-20180413-unofficial-condor": 1,
  "15.1-20180629-unofficial-condor": 2,
  "15.1-20200723-unofficial-condor": 1,
  "15.1-20200809-unofficial-condor": 1,
  "15.1-20200823-unofficial-condor": 31,
  "lineage_condor-userdebug 7.1.2 n2g47o 8c11d3f84c test-keys": 2
}