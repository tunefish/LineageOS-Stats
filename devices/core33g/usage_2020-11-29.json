{
  "13.0-20160827-unofficial-core33g": 5,
  "14.1-20170409-unofficial-core33g": 8,
  "14.1-20170820-unofficial": 89,
  "14.1-20170820-unofficial-core33g": 5,
  "14.1-20180615-unofficial-core33g": 179,
  "14.1-20180804-unofficial-core33g": 12,
  "14.1-20190421-unofficial-core33g": 2,
  "14.1-20190523-unofficial-core33g": 21,
  "14.1-20190609-unofficial-core33g": 3808,
  "14.1-20200802-unofficial-core33g": 1,
  "15.1-20180223-unofficial-core33g": 39,
  "15.1-20180708-unofficial-core33g": 78,
  "15.1-20180709-unofficial-core33g": 1,
  "15.1-20180711-unofficial-grandneove3g": 63,
  "17.1-20200715-unofficial-tissot": 6,
  "17.1-20200716-unofficial-herolte": 6,
  "17.1-20200821-unofficial-z3": 37
}