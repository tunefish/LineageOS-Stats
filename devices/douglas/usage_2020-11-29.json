{
  "12.1-20190913-unofficial-douglas": 2,
  "12.1-20190914-unofficial-douglas": 57,
  "12.1-20191026-unofficial-douglas": 70,
  "12.1-20200110-unofficial-douglas": 8,
  "12.1-20200227-unofficial-douglas": 1,
  "12.1-20200315-unofficial-douglas": 3,
  "12.1-20200325-unofficial-douglas": 620,
  "12.1-20200514-unofficial-douglas": 6,
  "12.1-20200614-unofficial-douglas": 40,
  "12.1-20200905-unofficial-douglas": 172,
  "14.1-20200926-unofficial-douglas": 1,
  "14.1-20201025-unofficial-douglas": 2,
  "14.1-20201103-unofficial-douglas": 54,
  "14.1-20201105-unofficial-douglas": 5,
  "14.1-20201113-unofficial-douglas": 3,
  "14.1-20201126-unofficial-douglas": 30,
  "14.1-20201128-unofficial-douglas": 10
}