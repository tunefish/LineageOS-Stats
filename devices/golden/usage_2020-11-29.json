{
  "13.0-20170118-unofficial-golden": 1,
  "13.0-20170121-unofficial-golden": 1,
  "13.0-20170129-unofficial-golden": 77,
  "13.0-20170224-unofficial-golden": 5,
  "13.0-20170302-unofficial-golden": 6,
  "13.0-20170310-unofficial-golden": 13,
  "13.0-20170328-unofficial-golden": 643,
  "13.0-20170420-unofficial-golden": 40,
  "14.1-20170122-unofficial-golden": 214,
  "14.1-20170318-unofficial-golden": 4,
  "14.1-20170402-unofficial-golden": 6,
  "14.1-20170415-unofficial-golden": 4,
  "14.1-20170419-unofficial-golden": 5,
  "14.1-20170507-unofficial-golden": 15,
  "14.1-20170522-unofficial-golden": 1,
  "14.1-20170609-unofficial-golden": 385,
  "14.1-20170610-unofficial-golden": 6,
  "14.1-20170628-unofficial-golden": 644,
  "14.1-20170724-unofficial-golden": 41,
  "14.1-20170817-unofficial-golden": 26,
  "14.1-20170920-unofficial-golden": 7,
  "14.1-20171011-unofficial-golden": 58,
  "14.1-20171215_192311-official-golden": 132,
  "14.1-20171215_192311-unofficial-golden": 671,
  "14.1-20180109_002536-unofficial-golden": 154,
  "14.1-20181014-unofficial-golden": 2,
  "14.1-20200817_071197-official-golden-rogerwed": 1,
  "15.1-20181030-unofficial-golden": 5,
  "17.1-20201113-unofficial-tissot": 1,
  "golden-user 8.1.0 oe632e 3de271a2ce release-keys": 1,
  "njh47f release-keys": 1
}